### DISCUSSION TITLE
MultiSelectFilter does show duplicates

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2485] [Help]
Hi all,

on a custom table I used a MultiSelectFilter that is based on a relationship:
```
            MultiSelectFilter::make('Fulfilment Status')
                ->relationship('fulfilment', 'fulfilment_status'),
```

In the table there are different entries that might have the same status, so status has many duplicates across the records.
Even though the table works it returns all duplicates as single entries:
![grafik](https://user-images.githubusercontent.com/67121752/169565960-3a54e3a5-7764-433e-83cb-91795f97890f.png)
Is there a way to get rid of all duplicates?

Thanks!

**Comment by danharrin** [comment]
There are many statuses with the same name? Not sure if this is a bug or something

**Reply by vv-ac** [reply]
No, there are many entries that have the same status, so like 20 entries which share 3 different status.
Then, when using the filter it shows the status 5 times, if 5 entries have that status and so on.

**Reply by danharrin** [reply]
Yeah that will be a bug then, please report

**Reply by synsyst** [reply]
Still an issue?

![image](https://github.com/user-attachments/assets/9a269ba4-82ee-4d17-9532-c31b0a0d625d)

`                SelectFilter::make('department')
                    ->label(__('Department'))
                    ->relationship('consultant.user.entra_user', 'department'),`
